cypress/support/pageObject/leavePage.ts

```diff
@@ -0,0 +1,33 @@
+type keyVal = {
+    key: string;
+    value: string;
+  };
```

Comment:
let's have the types in a different file, like you can have a `types.ts` file right?